src/liara.js

```diff
@@ -33,7 +33,12 @@ const liaraConfPath = join(homedir(), '.liara.json');
 
 let liaraConf;
 try {
-  liaraConf = JSON.parse(readFileSync(liaraConfPath));
+  if(!args.api_token){
```

Comment:
Still uses `api_token` with the underline.

Reply:
`api_token` is default property we get from `liaraConf` variable when user login successfully.